src/peergos/shared/user/UserContext.java

```diff
@@ -286,6 +286,35 @@ public boolean isJavascript() {
                 .thenApply(x -> globalRoot);
     }
 
+    @JsType
+    public static class EncryptedURL {
```

Comment:
Can we have a few unit tests for these, please? How about the classic (excuse the pseudocode):

```
x = EncryptedURL(a, b)
   assert x.decryptURLL(x.encryptURL(url)) == url
```